cypress/e2e/3-testing/workflow-ca.cy.js

```diff
@@ -0,0 +1,58 @@
+describe("Social Media App: Unauthorized user", () => {
+  beforeEach(() => {
+    cy.visit("https://nyolarraklay.github.io/social-media-client-CA-workflow");
```

Comment:
Its important to test the specific code in the current repo, not a hosted version. This may give a false positive if something is broken in the code.